README.md

```diff
@@ -1,18 +1,17 @@
 # **Final Project - Pneumonia identification from X-Ray**
 ### Big Data Engineering
 
-*by Yaiza ARNAIZ ALCACER, Pablo MARCOS LOPEZ*
+*by Yaiza ARNAIZ ALCACER and Pablo Ignacio MARCOS LOPEZ*
 
 ## Abstract
 
 Pneumonia is a serious illness characterised by a severe cough with phlegm, fever, chills and shortness of breath, which is caused by inflammation of the alveoli in one or both lungs. Despite advances in the diagnosis and treatment of lung infections, pneumonia is the sixth leading cause of death in adults in the United States, with more than six million cases of acute pneumonia each year, one million of which require hospitalisation.
 
 The simplest way to determine the extent and location of the infection, so that it can be treated more efficiently, is by taking chest X-rays; however, at present, analysis of these X-rays is limited to what humans are capable of, with the time delays and enormous costs that this entails.
 
-The aim of this paper is to present a complete model of thoracic X-ray image analysis, which we have developed by stacking two different Machine Leaning models, so that future scientists can build on it to hopefully improve the diagnostic lead times and economic efficiency of this class of tests.
+The aim of this paper is to present a complete model of thoracic X-ray image analysis, which we have developed by stacking two different Machine Leaning models, so that future scientists can build on it to hopefully improve the waiting time and economic efficiency of this class of tests.
 
-The results after training with two epochs is unexpected for the small loss obtained. The model implemented does not learn what is needed for an accurate classification. Nevertheless we configure a scalable model following the current state of the art and prove the transfer learning of more than one model stacking them together. As a follow up 
-experiment, the transfer learning can be still done with more than one model but instead stacking, a voting system could be implemented. 
+The results after training with two epochs is unexpected given the small loss obtained, and, at 68.26%, the implemented model is not excessively accurate when classifying. Nevertheless, the model we are presenting is scalable, shows state-of-the-art techniques, and proves that that transfered learning is possible even for more than one, stacked networks. It is up to future researchers (possibly for those with more ressources) to improve the stacking process by, for example implementing a voting system in the networks.
```

Comment:
A voting system is not a stacking process, this does not make sense at all, it is another technique of machine learning and you cannot say transferred learning, is transfer learning always, there is no way of changing that verb. I know because I read a lot of different papers and it is always transfer learning.